packages/lambdas/src/ihe.ts

```diff
@@ -0,0 +1,22 @@
+import * as Sentry from "@sentry/serverless";
+import { capture } from "./shared/capture";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const buildResponse = (status: number, body?: unknown) => ({
+  statusCode: status,
+  body,
+});
+
+const defaultResponse = () => buildResponse(200);
+
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Request = { body?: any; headers: Record<string, string> };
```

Comment:
Can't we use built-in ones, like `APIGatewayProxyEvent`?

Reply:
yup good point - will address